Tangem/Modules/Markets/TokenMarketsDetails/Subviews/PricePerformance/MarketsTokenDetailsPricePerformanceViewModel.swift

```diff
@@ -37,22 +37,27 @@ class MarketsTokenDetailsPricePerformanceViewModel: ObservableObject {
     }
 
     private func bind() {
-        currentPricePublisher
-            .combineLatest($selectedInterval)
-            .receive(on: DispatchQueue.main)
+        $selectedInterval
             .withWeakCaptureOf(self)
-            .sink(receiveValue: { value in
-                let weakSelf = value.0
-                let (currentPrice, interval) = value.1
-
+            .sink { viewModel, interval in
                 Analytics.log(
                     event: .marketsChartButtonPeriod,
                     params: [
-                        .token: weakSelf.tokenSymbol.uppercased(),
-                        .period: interval.rawValue,
+                        .token: viewModel.tokenSymbol.uppercased(),
+                        .period: interval.analyticsParameterValue,
                         .source: Analytics.MarketsIntervalTypeSourceType.price.rawValue,
                     ]
                 )
+            }
+            .store(in: &bag)
+
+        currentPricePublisher
+            .combineLatest($selectedInterval)
+            .receive(on: DispatchQueue.main)
+            .withWeakCaptureOf(self)
+            .sink(receiveValue: { value in
+                let weakSelf = value.0
```

Comment:
чтобы как и везде пишем
```suggestion
                let viewModel = value.0
```

Reply:
> чтобы как и везде пишем

Да окей, но мы вроде и так и так. Но на этом экране да, сделаю под один формат